src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int OPERATION_NAME = 0;
+    private static final int OPERATION_AMOUNT = 1;
+    private static final String REGEX = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String infoAboutBuyOperation = getSumOfOperation(readTheFile(fromFileName), Operation.BUY);
+        String infoAboutSupplyOperation = getSumOfOperation(readTheFile(fromFileName),
+                Operation.SUPPLY);
```

Comment:
overwrite your logic so you won't need to make two for loops for calculation of buy and supply